src/renderer/address-book/add/index.jsx

```diff
@@ -1,37 +1,188 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { AddressBookAdd, ModalBox } from 'selfkey-ui';
+import { StyledButton, ModalBox } from 'selfkey-ui';
 import { addressBookSelectors, addressBookOperations } from 'common/address-book';
+import { Grid } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+	errorText: {
```

Comment:
@eduardosmoura we will need to remove this css because the material UI theme we created will already have it. For instance for the error you can achieve it like this https://github.com/SelfKeyFoundation/Identity-Wallet/blob/dev/src/renderer/wallet/unlock/private-key.jsx#L122. As we are really close to the deadline I suggest we leave it for later and @designhorf can fix this pretty quick. Thoughts?

Reply:
Sorry, @rodrigopavezi. I didn't know about that. I think @designhorf is more agile than me resolving those CSS issues. Also, I showed him a black space at the of the address page main page. We will fix that later as well. Please, @designhorf let us know your thoughts on this matter? Thank you.